src/Content/Embed/style.ts

```diff
@@ -334,3 +334,8 @@ export const VideoThumbnailPlayButton = styled.withConfig({
     opacity: 1,
   },
 });
+
+export const Div = styled.withConfig({
```

Comment:
Is this used anywhere?

Reply:
Ah, I forgot to remove it. Let me do it.